### DISCUSSION TITLE
ROS2 Router: Service Call Request made, but no Response

### DISCUSSION THREAD

**Discussion by Thongtdextra** [eProsima/Fast-DDS#5159] [Q&A]
![image](https://github.com/user-attachments/assets/327ce89a-d7e7-440e-920a-1170a8ee4084)
I have 2 host in 1 LAN, HostA in domain 4 and have interface 20.13.21.128. HostB in domain 3 and have interface 20.13.21.24. 
In theory, if you want 2 host with differ domain can communicate each other then you have to setup ddsrouter in both host. 
I done with setup and 2 host can see topic each other but i have some problem with service.
In hostB, i run a server service, it can operated in local (if you run ros2 service call in local host <hostB - domain 3> then you can receive respone)
but if you run  this command: ros2 service call in <hostA-domain4> then hostB can take a request from hostA-client service but hostA can not receive a respone from hostB ? 
![image](https://github.com/user-attachments/assets/ed200dff-8b8a-47b1-a0ef-1966eee6e1d0)

**Comment by Thongtdextra** [comment]
i'm try charge qos of topic in ddsrouter file yaml . Structure of topic service have 2 topic request and respone, then i setup for above topic qos is best effort. in qos of server service and client service same qos. but i still stuck with this problem. summon that server can receive request from client but client cannot receive respone from server ?
![image](https://github.com/user-attachments/assets/a4ecbcb4-e12e-46e5-82c6-d99e6e79df1d)

**Comment by JesusPoderoso** [comment]
Duplicated thread, please refer to
- https://github.com/eProsima/DDS-Router/discussions/464
